### ISSUE TITLE
Conflict with Replaymod

### ISSUE THREAD

**Issue by Fizedi** [maruohon/tweakeroo#73]
Hi there,

I've discovered a conflict between tweakeroo and replaymod. When tweakaroo is loaded replay mod does not record gameplay. Replaymod menus are not visible either.

Hopefully it will be an easy fix and the 2 mods can get along. :) 

Thanks.

Fizedi

**Comment by InfoTeddy** [comment]
Which versions of Replaymod and Tweakeroo are you using?

**Comment by Fizedi** [comment]
Hi.
Here are the plugins I have installed:
fabric-api-0.5.1+build.294-1.15
litematica-fabric-1.15.2-0.0.0-dev.20200515.184506
malilib-fabric-1.15.2-0.10.0-dev.21+arne.2
minihud-fabric-1.15.2-0.19.0-dev.20200121.203515
optifabric-1.0.0-beta8
preview_OptiFine_1.15.2_HD_U_G1_pre13 (1)
replaymod-1.15.2-2.3.0
tweakeroo-fabric-1.15.2-0.10.0-dev.20200424.222527.jar

I noticed if I went in config folder and deleted the json files for both it started to work. But only for a short time, something happened and it messed up again. Not sure what the trigger was.

Also quick clarification question if I may. The feature to place blocks backwards (like pistons) is only achievable with a serverside plugin. Is that right or am I missing something? I am running PaperMC.

**Comment by InfoTeddy** [comment]
> Also quick clarification question if I may. The feature to place blocks backwards (like pistons) is only achievable with a serverside plugin. Is that right or am I missing something? I am running PaperMC.

Yes, you need [Carpet Mod](/gnembon/fabric-carpet) to let you place blocks with that sort of orientation. It's a Fabric mod, which I don't think can be installed on Paper servers.

**Comment by vacla** [comment]
Doesnt work on paper

**Comment by maruohon** [comment]
I need more information. Just having Tweakeroo installed (as well as all my other client mods) doesn't seem to break Replaymod. I was able to record a short replay and play it back.

If it crashes, then pastebin the crash log, otherwise you need to find out which Tweakeroo feature being enabled breaks Replaymod. I'm not going to test everything myself to try to find something that may be wrong somewhere.
